### PR TITLE
switch to lerna for multi-pkg mgmt

### PR DESCRIPTION
**Pull request by dbushong**
* rename linters/ to required packages/
* run `lerna init`
* update `lerna.json` version to `3.2.0`
* write a CONTRIBUTING file explaining publishing

TODO: actually test this

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+### Publishing changes
+
+(Notes for internal team)
+
+This multi-repository is managed with [lerna](https://github.com/lerna/lerna)
+2.x.
+
+```
+$ npm install --global lerna@^2.0.0-beta
```
Comment (jkrems) [inline CONTRIBUTING.md]: Since `lerna` is in the `devDependencies`, could this be just `npm install`?

File: test/compare-versions.test.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ var LINT_PKGS = [
   'eslint-config-groupon-react',
   'stylint-config-groupon',
 ].map(function (linter) {
-  return require('../linters/' + linter + '/package.json');
+  return require('../packages/' + linter + '/package.json');
```
Comment (jkrems) [inline test/compare-versions.test.js]: I think this test is expected to fail as soon as we publish our first package using lerna - correct?

Comment (dbushong) [inline test/compare-versions.test.js]: Good point.  Probably not necessary?

Comment (jkrems) [inline test/compare-versions.test.js]: Yeah, I think we yielded that concern to lerna with this PR.

**Pull request merged**